frontend/src/metabase/visualizations/components/TableInteractive/TableInteractive.jsx

```diff
@@ -115,6 +114,8 @@ class TableInteractive extends Component {
     this.headerRefs = [];
     this.detailShortcutRef = createRef();
 
+    this.gridRef = createRef();
```

Comment:
I changed this from a `findDOMNode` to a ref since I thought it was part of the issue, but it wasn't. I feel like this would still be good to have though.